src/util/db.js

```diff
@@ -112,10 +100,10 @@ async function storagePushValue(key, value) {
 }
 
 export async function loadToCAndPPConfirmation() {
-  const result = await AsyncStorage.getItem('ToCAndPPConfirmation');
+  const result = await AsyncStorage.getItem('ToCAndPPConfirmation_v3');
   return !!result;
 }
 
 export async function saveToCAndPPConfirmation() {
-  await AsyncStorage.setItem('ToCAndPPConfirmation', 'yes');
+  await AsyncStorage.setItem('ToCAndPPConfirmation_v3', 'yes');
```

Comment:
why is this called ToC btw? I don't think we need to change since we all know it means Terms and Conditions but just curious

Reply:
It's the flag to know if users have accepted the ToCs and our privacy policies (PP) and that will either tell the app to show the Account list view, or the ToCs screen.
So bumping it here allows us to :
- show the ToCs and PP screen again, even if users have accepted the v2 of those ToCs and PPs
- migrate v2 accounts if any

Reply:
yes, but..shouldn't it be `TaC` or `TnC`?

Reply:
didn't even realize it. I think it's an app wide typo!